packages/applications-service-api/prisma/migrations/20231129122831_remove_rep_redacted_fd/migration.sql

```diff
@@ -0,0 +1,25 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `redacted` on the `Representation` table. All the data in the column will be lost.
+
+*/
+BEGIN TRY
+
+BEGIN TRAN;
+
+-- AlterTable
+ALTER TABLE [dbo].[Representation] DROP COLUMN [redacted];
```

Comment:
You could combine the two migrations into one. The idea being to take this `ALTER TABLE` line and put it after the `DROP CONSTRAINT` in the other migration

Reply:
Good shout and much better merging into 1.